Daf.Meta/Model.cs

```diff
@@ -870,7 +915,7 @@ public static void Deserialize(string folder)
 					}
 				}
 
-				dataSource.StagingTable = new StagingTable();
+				StagingTable stagingTable = new();
```

Comment:
Is this never assigned to the data source?

Reply:
Nevermind, I see it now. If we make the stagingtable non-settable, this shouldn't be an issue, since it'll need to be refactored.

Reply:
Fixed the issues you commented on.

I will implement the same changes to LinkRelationships as are present in HubRelationships to keep the list of available columns updated. Then we can discuss the way I've subscribed to events and make sure they are correctly unsubscribed from when objects are removed. Then I should be able to submit a final draft for review. Thanks for the feedback!